app/players/elves/elf_ranger.py

```diff
@@ -0,0 +1,15 @@
+from app.players.elves.elf import Elf
+
+
+class ElfRanger(Elf):
+    def __init__(self, nickname: str, musical_instrument: str, bow_level: int):
+        super().__init__(nickname, musical_instrument)
+        self.__bow_level = bow_level
```

Comment:
It must be a protected value, not private.

Reply:
Thx ofr your cooment's!